lib/mailer/sfMailer.class.php

```diff
@@ -239,7 +239,9 @@ public function setDeliveryAddress($address)
    */
   public function compose($from = null, $to = null, $subject = null, $body = null)
   {
-    return Swift_Message::newInstance()
+    $msg = new Swift_Message();
```

Comment:
$msg is not needed :)
Why don't we `return new Swift_Message();` ?

Reply:
Well, because there are a few method calls on the object. That would mean to `return (new Swift_Message())->setFrom($from)...;`. Should I change it? Don't think it's mission critical, but would be ok for me.